convert2rhel/backup.py

```diff
@@ -348,6 +348,79 @@ def restore(self):
         super(RestorableRpmKey, self).restore()
 
 
+class NewRestorableFile(RestorableChange):
+    def __init__(self, filepath):
+        super(NewRestorableFile, self).__init__()
+        self.filepath = filepath
+
+    def enable(self):
+        """Save current version of a file"""
+        # Prevent multiple backup
+        if self.enabled:
+            return
+
+        loggerinst.info("Backing up %s." % self.filepath)
+        if os.path.isfile(self.filepath):
+            try:
+                loggerinst.debug("Copying %s to %s." % (self.filepath, BACKUP_DIR))
+                shutil.copy2(self.filepath, BACKUP_DIR)
+
```

Comment:
I'd probably move this around so that we have a debug message for successfully backing up the file after copying it, unless we do this somewhere else then consistency is better

Reply:
I'll change this in this PR which is related to that: https://github.com/oamg/convert2rhel/pull/935